server/db/driver/pg/internal/meta.go

```diff
@@ -3,11 +3,13 @@ package internal
 const (
 	// CreateMetaTable creates a table to hold DEX metadata.
 	CreateMetaTable = `CREATE TABLE IF NOT EXISTS %s (
```

Comment:
The table name is hardcoded below in `CreateMetaRow, SelectDBVersion, SetDBVersion`. Do we want to do the same here? I see from usage that the table name is always set to the constant `metaTableName     = "meta"`.

Reply:
Good call!  Hopefully no upgrades push what can be done in a single transaction.

Reply:
Maaaybe, but the only reason it's a placeholder in `CreateMetaTable` is to shoehorn it into the `tableMap` used by `CreateTable`, which expects each query to have this format with one specifier for the name of the table.  With meta in the public schema (default) and a name that isn't going to change, I'm inclined to keep the callign code for these other queries simple with no sprintf business.